third_party/libapr1.BUILD

```diff
@@ -99,7 +99,6 @@ cc_library(
         "tables/apr_tables.c",
         "threadproc/unix/proc.c",
         "threadproc/unix/procsup.c",
-        "threadproc/unix/signals.c",
```

Comment:
Can we restore this line? Removing it breaks importing the package on Linux due to `undefined symbol: apr_proc_kill`

Reply:
Updated